webpack.min.config.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+const webpack = require('webpack');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+
+module.exports = {
+  entry: './src/api/v4/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist/public'),
+    filename: 'carto.min.js',
+    library: 'carto',
+    libraryTarget: 'umd'
+  },
+  plugins: [
+    new UglifyJsPlugin(),
```

Comment:
I think the plugins are executed in the specified order so first we need to ignore the camshaft references and then uglify the whole bundle

Reply:
I think it doesn't matter, because the resulting carto.min.js has the same size. I'm going to change the order anyway.